### PR TITLE
Change links in spec.md and fix some errors in example

### PR DESCRIPTION
**Pull request by Torbacka**

### CODE REVIEW

File: example/se/PrivateUnsecuredLoanApplicationCreated.json [hunk 1]
```diff
@@ -75,9 +75,9 @@
 			"refinanceAmount": 0,
 			"termMonths": 24
 		},
-		"brokerReference": {
-			"issuer": "io.klira",
-			"id": "1"
+		"brokerReference":{
+			"issuer":"se.sverker",
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanApplicationCreated.json]: Why SE.sverker?

Comment (williamhogman) [reply example/se/PrivateUnsecuredLoanApplicationCreated.json]: com.example is more apt

File: example/se/PrivateUnsecuredLoanApplicationCreated.json [hunk 2]
```diff
@@ -5,17 +5,17 @@
...
     "data": {
 		"application": {
 			"applicant": {
-				"ssn": "198712167249",
-				"phone": "+46705431052",
+				"ssn": "19911216XXXX",
+				"phone": "+4670543100",
 				"secondaryPhone": [],
-				"emailAddress": "anton@zensum.se",
+				"emailAddress": "anton@sum.se",
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanApplicationCreated.json]: customer@example.com

File: example/se/PrivateUnsecuredLoanApplicationCreated.json [hunk 3]
```diff
@@ -5,17 +5,17 @@
...
+		"destination":"bank:bankname-se"
+	},
     "contentType" : "application/json",
     "data": {
 		"application": {
 			"applicant": {
-				"ssn": "198712167249",
-				"phone": "+46705431052",
+				"ssn": "19911216XXXX",
+				"phone": "+4670543100",
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanApplicationCreated.json]: Why?

File: example/se/PrivateUnsecuredLoanApplicationCreated.json [hunk 4]
```diff
@@ -5,17 +5,17 @@
...
+	"extensions":{
+		"destination":"bank:bankname-se"
+	},
     "contentType" : "application/json",
     "data": {
 		"application": {
 			"applicant": {
-				"ssn": "198712167249",
-				"phone": "+46705431052",
+				"ssn": "19911216XXXX",
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanApplicationCreated.json]: Don't do XXXX but an actual fake SSN in there

File: example/se/PrivateUnsecuredLoanApplicationCreated.json [hunk 5]
```diff
@@ -5,17 +5,17 @@
     "source" : "/mycontext",
     "eventID" : "C234-1234-1234",
     "eventTime" : "2018-04-05T17:31:00Z",
-    "extensions" : {
-      "comExampleExtension" : "value"
-    },
+	"extensions":{
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanApplicationCreated.json]: Not part of the spec, it shouldn't be there

File: example/se/PrivateUnsecuredLoanOfferAccepted.json [hunk 6]
```diff
@@ -5,14 +5,14 @@
...
+  "extensions":{
+    "destination":"bank:bankname-se"
   },
   "contentType" : "application/json",
   "data": {
-    "brokerReference": {
-      "id": "14-0e2375ac997e17a54c1106a7e120fcf3b18a59de923bf4b5d14a61aea93a2a6a",
-      "issuer": "io.klira"
+    "brokerReference":{
+      "issuer":"se.sverker",
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanOfferAccepted.json]: See above

File: example/se/PrivateUnsecuredLoanOfferAccepted.json [hunk 7]
```diff
@@ -5,14 +5,14 @@
   "source" : "/mycontext",
   "eventID" : "C234-1234-1234",
   "eventTime" : "2018-04-05T17:31:00Z",
-  "extensions" : {
-    "comExampleExtension" : "value"
+  "extensions":{
+    "destination":"bank:bankname-se"
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanOfferAccepted.json]: See above

File: example/se/PrivateUnsecuredLoanOfferRejected.json [hunk 8]
```diff
@@ -10,9 +10,9 @@
   },
   "contentType" : "application/json",
   "data": {
-    "brokerReference": {
-      "id": "14-0e2375ac997e17a54c1106a7e120fcf3b18a59de923bf4b5d14a61aea93a2a6a",
-      "issuer": "io.klira"
+    "brokerReference":{
+      "issuer":"se.sverker",
+      "id":"2-147625e89efdea1c9ddaa96d29592920c485dd2e153f4f4ca49beb81e3"
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanOfferRejected.json]: See above

File: example/se/PrivateUnsecuredLoanOffering.json [hunk 9]
```diff
@@ -26,14 +26,13 @@
...
-      "nominalInterestRate": "sdfs",
       "offeredCredit": 123456,
       "termFee": 1234,
       "termMonths": 70
     },
-    "brokerReference": {
-      "id": "14-0e2375ac997e17a54c1106a7e120fcf3b18a59de923bf4b5d14a61aea93a2a6a",
-      "issuer": "io.klira"
+    "brokerReference":{
+      "issuer":"se.sverker",
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanOffering.json]: See above

File: example/se/PrivateUnsecuredLoanRejection.json [hunk 10]
```diff
@@ -10,9 +10,9 @@
   },
   "contentType" : "application/json",
   "data": {
-    "brokerReference": {
-      "id": "14-0e2375ac997e17a54c1106a7e120fcf3b18a59de923bf4b5d14a61aea93a2a6a",
-      "issuer": "io.klira"
+    "brokerReference":{
+      "issuer":"se.sverker",
+      "id":"2-147625e89efdea1c9ddaa96d29592920c485dd2e153f4f4ca49beb81e3"
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanRejection.json]: See  above

File: example/se/PrivateUnsecuredLoanStatusUpdated.json [hunk 11]
```diff
@@ -11,9 +11,9 @@
   "contentType" : "application/json",
   "data": {
     "delayReason": "MANUAL_PROCESSING",
-    "brokerReference": {
-      "issuer": "io.klira",
-      "id": "1"
+    "brokerReference":{
+      "issuer":"se.sverker",
+      "id":"2-147625e89efdea1c9ddaa96d29592920c485dd2e153f4f4ca49beb81e3"
```
Comment (williamhogman) [inline example/se/PrivateUnsecuredLoanStatusUpdated.json]: see above

File: example/se/PrivateUnsecuredLoanApplicationCreated.json (hunk 3 above)
Comment (williamhogman) [reply example/se/PrivateUnsecuredLoanApplicationCreated.json]: SSN with XXXX is also invalid per the spec

File: example/se/PrivateUnsecuredLoanRejection.json (hunk 10 above)
Comment (Torbacka) [reply example/se/PrivateUnsecuredLoanRejection.json]: Is broker reference not a part of the specification? Or what do you mean with see above?

File: example/se/PrivateUnsecuredLoanApplicationCreated.json (hunk 5 above)
Comment (Torbacka) [reply example/se/PrivateUnsecuredLoanApplicationCreated.json]: i'm quite sure its part of cloud event specification https://github.com/cloudevents/spec/blob/v0.1/spec.md#extensions.

The example should showcase how a PrivateUnsecruredLoanApplicationCreated event looks like. Right now all events of that type has an attribute called extensions with another attribute called destination.

File: example/se/PrivateUnsecuredLoanOfferAccepted.json (hunk 7 above)
Comment (Torbacka) [reply example/se/PrivateUnsecuredLoanOfferAccepted.json]: See above

File: example/se/PrivateUnsecuredLoanOfferRejected.json (hunk 8 above)
Comment (Torbacka) [reply example/se/PrivateUnsecuredLoanOfferRejected.json]: See above

Comment (Torbacka) [reply example/se/PrivateUnsecuredLoanOfferRejected.json]: Above is refereeing to a separate attribute. Also m quite sure brokerReference is part of the spec.

**Comment by Torbacka** [comment]
Close this pull request

**Pull request closed without merging**